scenarios/cloud_breach_s3/terraform/variables.tf

```diff
@@ -1,32 +1,53 @@
 #Required: AWS Profile
 variable "profile" {
-
+  description = "The AWS profile to use."
+  type        = string
 }
+
 #Required: AWS Region
 variable "region" {
-  default = "us-east-1"
+  description = "The AWS region to deploy resources to."
+  default     = "us-east-1"
+  type        = string
 }
+
 #Required: CGID Variable for unique naming
 variable "cgid" {
-
+  description = "CGID variable for unique naming."
+  type        = string
 }
+
 #Required: User's Public IP Address(es)
 variable "cg_whitelist" {
-  default = "../whitelist.txt"
+  description = "User's public IP address, pulled from the file ../whitelist.txt"
+  default     = ["0.0.0.0/0"]
```

Comment:
All IPs by default?

Reply:
We could switch this to something else (`0.0.0.0/32`) Its more to switch to a default value that would be [valid](https://github.com/RhinoSecurityLabs/cloudgoat/pull/214/files/8c62eaae8bde1a01f5d2d0b48d1dd43455dfce75#diff-4bc912b2516abb43f424509cf93dad506c7da00b4d39dc0c9fa26c19309c7cf5R46) (`list(string)`)